asab/library/providers/filesystem.py

```
@@ -78,24 +72,327 @@ def __init__(self, library, path, layer, *, set_ready=True):
+
+		for item in items:
+			if item.type == "dir":
+				self._subscribe_recursive(subscribed_path, item.name)
+
+
+	def _on_inotify_read(self):
+		data = os.read(self.FD, 64 * 1024)
+
+		pos = 0
+		while pos < len(data):
+			wd, mask, cookie, namesize = struct.unpack_from(EVENT_FMT, data, pos)
```

Ruff (0.15.12)

[warning] Unpacked variable `cookie` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)